site/content/api/Tenant-API.md

```diff
@@ -252,7 +252,7 @@ The table below provides an overview of the standard members defined for the JSO
 | :------------------------| :-------: | :--------- | :------------ | :---------- |
 | *tenant-id*              | *yes*     | *string*   |               | The ID of the tenant. |
 | *enabled*                | *no*      | *boolean*  | `true`        | If set to `false` the tenant is currently disabled. Protocol adapters MUST NOT allow devices of a disabled tenant to connect and MUST NOT accept data published by such devices.
-| *adapters*               | *no*      | *array*    | empty array   | A list of configuration options valid for certain adapters only. See [Adapter Configuration Format]({{< relref "#adapter-configuration-format" >}}) for details. **NB** Only a single entry per *type* is allowed. If multiple entries for the same type are present it is handled as an error.
+| *adapters*               | *no*      | *array*    | empty array   | A list of configuration options valid for certain adapters only. The format of a configuration option is described here [Adapter Configuration Format]({{< relref "#adapter-configuration-format" >}}). **NB** Only a single entry per *type* is allowed. If multiple entries for the same type are present it is handled as an error. **NB** If for a certain protocol adapter no configuration is present it has the same semantic as a configuration present with default values.
```

Comment:
> If for a certain protocol adapter no configuration is present it has the same semantic as a configuration present with default values.

I am not sure if this the default behavior we want. My understanding is that the whole point in enabling adapters at the tenant level is that the normal case will be that a tenant is *not* allowed to use all adapters, i.e. when I create a new tenant, I will usually also enable the one or two adapters (along with additional configuration) that it is allowed to use by default. However, the current approach would mean that we instead need to explicitly *disable* all protocol adapters that the tenant is not allowed to use **and** also configure the ones that it is allowed to use.

FMPOV it would make more sense to follow the following logic:
if the *adapters* element is omitted, then *all* adapters are allowed/enabled in their default configuration. However, if the element is provided then the list MUST NOT be empty and contains non-standard adapter configuration. I would then propose to change the default value of the adapter's *enabled* property to `false` so that you need to explicitly enable an adapter for which you provide explicit configuration. This way, you only need to include configuration for those adapters that the tenant is actually allowed to use when you create the tenant.

WDYT?

Reply:
That improves it for me.
So, with less words we have:
- *adapters* omitted -> all protocol adapters enabled with default values
- *adapter* provided -> non empty list, desired protocol adapters need to be configured, other protocol adapters are disabled

I have a good feeling about that approach matching the usual cases with tenant creation.

BUT: It can be a little confusing FMPOV for readers regarding the default value of *enabled*:
Actually I think, we do not need the *enabled* property with that logic anymore.
Why not removing this property?
Or do we need to configure protocol adapters, and then temporary switch them off (and on again) without loosing the config?
WDYT?

Reply:
Yep that approach seems legit to me too. But I would like to keep the `enabled` property, it's more like in the other apis and i like the possibility to quickly disable an adapter while testing something.

Reply:
I see your point, and it seems handy for testing. 

So, I propose to add a second example at the end to clarify the handling of adapter configuration.
I guess that is probably the last open point from my side :-)

Reply:
I added an example to clarify and worked the changes into it